content/guides/mobile/ios/quick-start/3-configure-box-app.md

```diff
@@ -0,0 +1,82 @@
+---
+type: quick-start
+hide_in_page_nav: true
+---
+
+# Configure a Box App
+
+To start making authenticated API calls to the Box API with the **Box iOS
+SDK**, an **Access Token** will be needed. The simplest way to generate a valid
+token is to generate a new **Box App** and manually generate a short lived
+developer token.
+
+The developer token is generated through the developer console UI and will be
+valid for one hour before having to be manually refreshed.
+
+## Set up a Box app
+
+<Grid columns='2'>
+  <Choose option='ios.app_type' value='create_new' color='blue'>
+    # Create a new Box app
+
+    Create and configure a new Box JWT application from which a developer
+    token may be generated
+  </Choose>
+
+  <Choose option='ios.app_type' value='use_own' color='red'>
+    # Use an existing app
+
+    Use one of your existing Box JWT applications
+  </Choose>
+</Grid>
+
+<Choice option='ios.app_type' value='use_own' color='blue'>
+  # Create a new Box app
+
+  To create a new Box application that may be used to generate a developer
+  token, use the following steps.
+
+  1. Go to the [Developer Console][devconsole]
+  1. Select **Create New App**
+  1. Select **Custom App** as the type of application to create, and click
+     **Next**
+  1. Select **OAuth 2.0 with JWT** as the authentication method, and click
+     **Next**
+  1. Give your Box app a unique name and click **Create App**
+  1. Go to the app's configuration by clicking **View Your App**.
+  1. Optionally, scroll to the **Application Scopes** section of the same screen
+     and select any additional permissions you want to enable for this
+     application.
+  1. At the top of the page click the button to **Save Changes**
+</Choice>
+
+<Choice option='ios.app_type' value='use_own' color='blue'>
+  # Use an Existing JWT Box application
+
+  If you have an existing JWT based Box application in your
+  [developer console][devconsole] that you
+  would like to use, skip to the next step.
+</Choice>
```

Comment:
Ok maybe we do need a `Choice` here. Maybe we might want to, expand here to explain how to generate a JWT? They might have created an app already but don't have a JWT keypair yet?

Reply:
I'm not sure that we want to go down that far - I wanted to try to avoid as much app configuration as possible as I felt it detracted from the purpose of the quickstart. Happy to revisit if you feel strongly otherwise though.